prometheus/value.go

```diff
@@ -153,11 +191,17 @@ func populateMetric(
 	labelPairs []*dto.LabelPair,
 	e *dto.Exemplar,
 	m *dto.Metric,
+	createdTimestamp *time.Time,
 ) error {
+	var ct *timestamppb.Timestamp
```

Comment:
A little bit more readable code is to add this in m.Counter.CreatedTimestamp ... once m.Counter is created.